sound/effect.h

```diff
@@ -352,11 +352,21 @@ class Effect {
 
 #ifdef NO_REPEAT_RANDOM
   int last_;
+  int last_sub_;
```

Comment:
we should make last_ and last_sub_  int16_t

Reply:
updated